src/strife/m_menu.c

```diff
@@ -559,9 +560,13 @@ void M_ReadSaveStrings(void)
             LoadMenu[i].status = 0;
             continue;
         }
-        fread(savegamestrings[i], 1, SAVESTRINGSIZE, handle);
+        retval = fread(savegamestrings[i], 1, SAVESTRINGSIZE, handle);
         fclose(handle);
+        if (retval == 0) {
+            LoadMenu[i].status = 0;
+        } else {
         LoadMenu[i].status = 1;
```

Comment:
Please pay attention to indentation.

Reply:
The indentation is already broken in some files and I dislike putting whitespace and functional changes in the same commit. This one could slip past but some places where bigger changes are needed can become impossible to read if they're mixed.

Reply:
Indentation should be correct in most cases, so I'm not sure what you mean. Have you set your editor to show tabs correctly as 8 spaces?

Reply:
At least one file was mixing tabs and spaces in the same function. That's wrong no matter what. I'll check the indentation when I revise this.

1. Are you ok with mixing up functional and whitespace changes?
2. Should a whitespace-only commit come before or after functional changes?

Reply:
> At least one file was mixing tabs and spaces in the same function. That's wrong no matter what.

As long as it displays correctly in a properly-configured editor, that should be fine.

Look, I appreciate it's not ideal and I admire your attention to detail. I'm careful to avoid mixing tabs and spaces when writing new code, but there's a lot of old code in the codebase and it's probably too much of a hassle to change.

> Are you ok with mixing up functional and whitespace changes?

No, please avoid whitespace-only changes if they're not necessary. However, if you're changing lines for functional reasons and want to expand the tabs, that's fine.

> Should a whitespace-only commit come before or after functional changes?

I'd rather avoid whitespace-only commits entirely if possible.